exercises/04_structs/structs2.nr

```diff
@@ -5,19 +5,19 @@
 // I AM NOT DONE
 
 struct Package {
-    sender_country: [u8],
-    recipient_country: [u8],
+    sender_country: [u8; 3],
+    recipient_country: [u8; 3],
     weight_in_grams: u32,
 }
 
 trait PackageTrait {
-    fn new(sender_country: [u8], recipient_country: [u8], weight_in_grams: u32) -> Self;
+    fn new(sender_country: [u8; 3], recipient_country: [u8; 3], weight_in_grams: u32) -> Self;
     fn is_international(self) -> //???;
     fn get_fees(self, cents_per_gram: u32) -> //???;
 }
 
 impl PackageTrait for Package {
-    fn new(sender_country: [u8], recipient_country: [u8], weight_in_grams: u32) -> Self {
+    fn new(sender_country: [u8; 3], recipient_country: [u8; 3], weight_in_grams: u32) -> Self {
         assert(weight_in_grams == 0);
```

Comment:
```suggestion
        assert(weight_in_grams != 0);
```
Can you also add the != 0 ? Even if Andy planned to do it, it is small enough so you can steal his work